### PR TITLE
Create insert query animals

### PR DESCRIPTION
**Pull request by evillaz**
This branch's purpose was to fulfill the following:

- Create a table named animals with the specified columns and copy the code into the schema.sql file
- Insert data by using the INSERT command and copy them into the data.sql file
- Write queries for the specified quests and copy them into the queries.sql file
- Take a screenshot of your queries' results and attach it to the pull request.
results

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+  id INTEGER,
+  name VARCHAR(255),
+  date_of_birth DATE,
+  escape_attempts INTEGER,
+  neutered BIT,
```
Comment (bobb-Rob) [inline schema.sql]: - [ ] Based on the project requirements, `neutered` should be of type `boolean -true or false` and not `BIT`. Kindly update this.

File: data.sql [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+VALUES (1, 'Agumon', '2020-02-03', 0, 1, 10.23);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (2, 'Gabumon', '2018-11-15', 2, 1, 8);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (3, 'Pikachu', '2021-01-07', 1, 0, 15.04);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (4, 'Devimon', '2017-05-12', 5, 1, 11);
```
Comment (bobb-Rob) [inline data.sql]: - [ ] As I said earlier about the data type `neutered`, it should be boolean - `true or false`. Please ensure you fix this

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @evillaz, 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of Gitflow
✅ Very descriptive Pull Request(PR)

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by evillaz** [comment]
I'm using MSSMS and Boolean data type is not included in SQL Server. So I used the bit data type that stores 0, 1, and NULL values that can be used instead of the TRUE, FALSE, and NULL values. What should I do then?

**Comment by evillaz** [comment]
I used MySQL to implement changes

File: data.sql [hunk 3]
```diff
@@ -1,5 +1,13 @@
...
+VALUES (1, 'Agumon', '2020-02-03', 0, true, 10.23);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (2, 'Gabumon', '2018-11-15', 2, true, 8);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (3, 'Pikachu', '2021-01-07', 1, false, 15.04);
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (4, 'Devimon', '2017-05-12', 5, true, 11);
```
Comment (amon-cofie) [inline data.sql:13]: [OPTIONAL] You have done well implementing the changes required 👍. Optionally, you could make the codebase DRY by using one ```INSERT``` statement here and making the code look something like this: 

```sh
  INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg)
  VALUES 
  (1, 'Agumon', '2020-02-03', 0, true, 10.23), 
  (2, 'Gabumon', '2018-11-15', 2, true, 8), 
  (3, 'Pikachu', '2021-01-07', 1, false, 15.04), 
  (4, 'Devimon', '2017-05-12', 5, true, 11);
```

**Comment by amon-cofie** [comment]
> I'm using MSSMS and Boolean data type is not included in SQL Server. So I used the bit data type that stores 0, 1, and NULL values that can be used instead of the TRUE, FALSE, and NULL values. What should I do then?

Oh okay, I know very little about SSMS but maybe you should consider installing PostgreSQL and using the pgAdmin 4 interface if you prefer not to use the CLI. While using ```BIT``` may work in this case I think it could potentially be a pain to solve if it causes certain semantic errors along the line, especially if you're working with someone.

**Review by amon-cofie — APPROVED** [review]
Hi @evillaz , 👋 

Your project is complete and very impressive 💯!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

🚀 You have a descriptive pull request summary
🚀 You have good commit messages
🚀 You used Gitflow properly


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**